src/main/java/seedu/mtracker/filemanager/Storage.java

```diff
@@ -0,0 +1,38 @@
+package seedu.mtracker.filemanager;
+
+import seedu.mtracker.model.Instrument;
+import seedu.mtracker.model.InstrumentManager;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+
+public class Storage {
+    private final File file;
+    public static final String FILE_PATH = "data/mTracker.txt";
+    private final Path path;
+
+    public Storage() {
+        file = new File(FILE_PATH);
+        path = Paths.get(FILE_PATH);
+    }
+
+    public void loadFileData(InstrumentManager instrumentManager) throws IOException {
+        if (!Files.exists(path) || !Files.isRegularFile(path)) {
+            file.getParentFile().mkdir();
+            file.createNewFile();
+            return;
+        }
```

Comment:
Can a add loadfile display here so that user knows that its loading from saved data. Under the if block add a create file display